part-overview.tex

```diff
@@ -60,7 +60,7 @@ \section{Application architecture}
 \begin{itemize}
 \item The web browser
 \item The web server (as a network traffic handler)
-\item The web server (as an application - CGI)
+\item The web server (as an application - \gls{cgi})

```

Comment:
I think it's better to mention PSGI rather than CGI these days, don't you think? I mean, PSGI can be plugged as CGI (but the startup time of our application is so long that it no longer fits the CGI single-shot execution paradigm...)

Reply:
Replaced all occurrences of cgi with psgi. Will be in the next PR.